examples/query_lang/multi_credentials.json

```diff
@@ -0,0 +1,33 @@
+{
+  "credentials": [
+    {
+      "id": "pid",
+      "format": "vc+sd-jwt",
+      "meta": {
+        "vct_values": ["https://credentials.example.com/identity_credential"]
+      },
+      "claims": [
+        {"path": ["given_name"]},
+        {"path": ["family_name"]},
+        {"path": ["address", "street_address"]}
+      ]
+    },
+    {
+      "id": "mdl",
+      "format": "mso_mdoc",
+      "meta": {
+        "doctype_values": ["org.iso.7367.1.mVR"]
+      },
+      "claims": [
+        {
+          "namespace": "org.iso.7367.1",
+          "claim_name": "vehicle_holder"
```

Comment:
While this is a non-normative example, is it confusing to include claims that are not mso_mdoc?